src/main/java/resources/TwitterRequestResource.java

```diff
@@ -22,51 +21,86 @@
     public final String VERSION = "1.0";
     private Twitter twitter;
     private ExceptionHandler exceptionHandler;
-    private static Logger LOGGER = LoggerFactory.getLogger(ErrorMessage.class);
+    private static Logger logger = LoggerFactory.getLogger(TwitterRequestResource.class);
 
     // CONSTRUCTORS
+    /** Constructor, sets up twitter using default configuration settings and exceptionHandler to a new ExceptionHandler
+     * */
     public TwitterRequestResource() {
-        Configuration c = new Configuration();
+        ApplicationConfiguration c = new ApplicationConfiguration();
         twitter = c.createTwitter();
         exceptionHandler = new ExceptionHandler();
+        logger.debug("TwitterRequestResource created with empty constructor");
     }
 
+    /** Constructor
+     * @param twitter Twitter to be set at the new twitter property
+     * @param exceptionHandler ExceptionHandler to be set as the new exceptionHandler property
+     * */
     public TwitterRequestResource(Twitter twitter, ExceptionHandler exceptionHandler) {
         this.twitter = twitter;
         this.exceptionHandler = exceptionHandler;
+        logger.debug("TwitterRequestResource created with twitter = {} and exceptionHandler = {}", twitter.toString(), exceptionHandler.toString());
     }
+
     // GETTERS
+    /** @return Twitter. The object's twitter property.
+     * */
     public Twitter getTwitter(){
+        logger.debug("getTwitter() was called to return {}", this.twitter.toString());
         return this.twitter;
     }
 
+    /** @return ExceptionHandler. The object's exceptionHandler property.
+     * */
     public ExceptionHandler getExceptionHandler(){
+        logger.debug("getExceptionHandler() was called to return {}", this.exceptionHandler.toString());
         return this.exceptionHandler;
     }
+
     // API CALLS
+    /** API call on /timeline
+     * @return Response. Contains list of statuses if successful, or error message if not.
+     * */
```

Comment:
i see that you looked up multi-line comments btw. 

but these don't seem to be providing anything helpful imo.

Reply:
I wanted to make sure that every function has a mention in the JavaDoc. Admittedly, I don't know if the team uses JavaDoc at all, so if we don't I could go ahead and remove most of the JavaDoc comments i made

Reply:
JavaDocs are useful when you need them. For instance, a complex method that is doing implementing core logic. Helping with understanding the thought process there is a useful thing. 

My comment here is more around the repetition of things. For instance
```
// API CALLS
/** API call .... 

```

Reply:
i see. I'll fix that